### ISSUE TITLE
Add Persian version of website pages

### ISSUE THREAD

**Issue by Metal-Bat** [open-telemetry/opentelemetry.io#4990]
**Desired feature or idea**: What would you like to see happen?

i like iranian people learn to use this project more and for that they need easier access so i hope i can be a help

**Comment by chalin** [comment]
From https://github.com/open-telemetry/opentelemetry.io/pull/4978#issuecomment-2273281303:

> heads up that Docsy (and hence this repo) doesn't fully support RTL languages yet. I'm working on it, but the feature might not land for a few weeks or months -- though I do see it happening before the end of the year. Reference: [google/docsy#1442](https://github.com/google/docsy/issues/1442).

**Comment by chalin** [comment]
We have made progress on Docsy support for RTL languages.

@Metal-Bat - could you take a look at https://example.docsy.dev/fa/docs/ and let us know if there are any layout problems. (We know about the [i18n] tags not yet being defined, those will be addressed via https://github.com/google/docsy/issues/2019).

If you find any issues, please add a note to https://github.com/google/docsy/issues/1442. Thanks!

**Comment by chalin** [comment]
Once the following PR is merged, this issue will be unblocked:

- #5442

**Comment by chalin** [comment]
#5442 has been merged, FYI.

**Comment by chalin** [comment]
FYI, great talk by @adowair: [What's in a Word? Lessons from Localizing Kubernetes Documentation to Arabic](https://youtu.be/HY3LZOQqdig?si=eGzIFf2uCxHSxERg).

**Comment by svrnm** [comment]
@Metal-Bat our localization efforts have grown and we are now able to support you, please take a look here:

https://opentelemetry.io/docs/contributing/localization/#new-localizations

If you want to rally people around a persian localization, let us know!

**Comment by Metal-Bat** [comment]
Thanks for the opportunity.
First, I need to find some people willing to contribute, then I will inform you.

**Comment by svrnm** [comment]
@Metal-Bat that's good to hear, we also can reach out to some folks to see if they are interested

**Comment by Metal-Bat** [comment]
Hi there,
sorry it took a little while.

We found some volunteers, but there is one issue: we currently don't have a mentor.
We would really appreciate it if someone could help introduce or suggest one.

Here are the folks who are willing to help:
@NeginEm98 @AfshinPardeyaghoot @HeliaMoazedy @moeinkolivand @AminAzarpey @shaqayegh-gh

If everything looks solid and okay or not okay, please let me know so I can fix it or start the process based on [localization](https://opentelemetry.io/docs/contributing/localization/#new-localizations) docs.

**Comment by svrnm** [comment]
@Metal-Bat and all, apologies for not following up properly on this, I was (and still am..) under water, but I am really excited about your interest here, I wanted to figure out if we can find a mentor for you among our maintainers and or the larger CNCF ecosystem

**Comment by Metal-Bat** [comment]
Hi there sorry for being late
trying to stay alive right now, in this shit hole, it seems we are getting connected super slowly (still using some weird vpn protocols) but still so hard to stay connected
now i need to make sure my friends are still alive and alright
I will be waiting for update
Thanks a lot

**Comment by svrnm** [comment]
@Metal-Bat (and all other contributors): we maintainers can not imagine what you are going through right now, and hope for your and your loved ones life and health! 

We understand that this here is not a priority for you right now, but want to provide all the support and assistance needed to move forward, if you ever want to. 

@theletterf offered to be the mentor for the Persian localization project, and based on the big support from folks, from our perspective we can get started. So, if you can provide a PR for localizing the landing page of the project, we can set you up as localization team.
